sdks/python/apache_beam/runners/dataflow/dataflow_job_service_test.py

```diff
@@ -56,5 +56,26 @@ def test_dry_run(self):
       job_servicer.stop()
 
 
+@unittest.skipIf(apiclient is None, 'GCP dependencies are not installed')
+class DirectPipelineTemplateTest(unittest.TestCase):
+  def test_template(self):
+    job_servicer = local_job_service.LocalJobServicer(
+        None, beam_job_type=dataflow_job_service.DataflowBeamJob)
+    port = job_servicer.start_grpc_server(0)
+    try:
+      options = PipelineOptions(
+          runner='PortableRunner',
+          job_endpoint=f'localhost:{port}',
+          project='some_project',
+          temp_location='gs://bucket/dir',
```

Comment:
@damccorm is there any convention on what to put here? otherwise this will just always fail

Reply:
Could you try the pipeline options here https://github.com/apache/beam/blob/5862bab9a275daaddafee911a897f4188ff1885c/sdks/python/apache_beam/runners/dataflow/template_runner_test.py#L54 - it won't work for you locally, but I think it should work in the CI environment which has some basic permissions set up for our `apache-beam-testing` project. If that doesn't work we can just revert and not worry about this test, the blast radius is relatively small anyways.